project/Dependencies.scala

```diff
@@ -95,21 +95,24 @@ object Dependencies {
-  private val akkaClusterTyped         = "com.typesafe.akka" %% "akka-cluster-typed" % Versions.Akka
-  private val akkaClusterShardingTyped = "com.typesafe.akka" %% "akka-cluster-sharding-typed" % Versions.Akka
-  private val akkaClusterTools         = "com.typesafe.akka" %% "akka-cluster-tools" % Versions.Akka
-  private val akkaDistributedData      = "com.typesafe.akka" %% "akka-distributed-data" % Versions.Akka
-  private val akkaJackson              = "com.typesafe.akka" %% "akka-serialization-jackson" % Versions.Akka
-  private val akkaMultiNodeTestkit     = "com.typesafe.akka" %% "akka-multi-node-testkit" % Versions.Akka
-  private val akkaPersistence          = "com.typesafe.akka" %% "akka-persistence" % Versions.Akka
-  private val akkaPersistenceQuery     = "com.typesafe.akka" %% "akka-persistence-query" % Versions.Akka
-  private val akkaSlf4j                = ("com.typesafe.akka" %% "akka-slf4j" % Versions.Akka).excludeAll(excludeSlf4j: _*)
-  private val akkaStream               = "com.typesafe.akka" %% "akka-stream" % Versions.Akka
-  private val akkaProtobuf_v3          = "com.typesafe.akka" %% "akka-protobuf-v3" % Versions.Akka
+  private val akkaActor            = "com.typesafe.akka" %% "akka-actor" % Versions.Akka
+  private val akkaRemote           = "com.typesafe.akka" %% "akka-remote" % Versions.Akka
+  private val akkaCluster          = "com.typesafe.akka" %% "akka-cluster" % Versions.Akka
+  private val akkaClusterSharding  = "com.typesafe.akka" %% "akka-cluster-sharding" % Versions.Akka
+  private val akkaClusterTools     = "com.typesafe.akka" %% "akka-cluster-tools" % Versions.Akka
+  private val akkaDistributedData  = "com.typesafe.akka" %% "akka-distributed-data" % Versions.Akka
+  private val akkaJackson          = "com.typesafe.akka" %% "akka-serialization-jackson" % Versions.Akka
+  private val akkaMultiNodeTestkit = "com.typesafe.akka" %% "akka-multi-node-testkit" % Versions.Akka
+  private val akkaPersistence      = "com.typesafe.akka" %% "akka-persistence" % Versions.Akka
+  private val akkaPersistenceQuery = "com.typesafe.akka" %% "akka-persistence-query" % Versions.Akka
+  private val akkaSlf4j            = ("com.typesafe.akka" %% "akka-slf4j" % Versions.Akka).excludeAll(excludeSlf4j: _*)
+  private val akkaStream           = "com.typesafe.akka" %% "akka-stream" % Versions.Akka
+  private val akkaProtobuf_v3      = "com.typesafe.akka" %% "akka-protobuf-v3" % Versions.Akka
+  // akka typed dependencies
+  private val akkaClusterTyped              = "com.typesafe.akka" %% "akka-cluster-typed"          % Versions.Akka
+  private val akkaPersistenceTyped          = "com.typesafe.akka" %% "akka-persistence-typed"      % Versions.Akka
+  private val akkaClusterShardingTyped      = "com.typesafe.akka" %% "akka-cluster-sharding-typed" % Versions.Akka
+  private val akkaJacksonSerializationTyped = "com.typesafe.akka" %% "akka-serialization-jackson"  % Versions.Akka
```

Comment:
I think `akka-serialization-jackson` is unrelated to `typed` and more general purpose, right?

Reply:
```suggestion
  private val akkaJacksonSerialization = "com.typesafe.akka" %% "akka-serialization-jackson"  % Versions.Akka
```

Meh, this suggestion will break formatting of previous lines because the name of this variable was pushing the `=` sign forward (affecting up to 4 lines above).